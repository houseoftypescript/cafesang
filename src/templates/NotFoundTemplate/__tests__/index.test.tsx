import { render } from '@testing-library/react';
import { NotFoundTemplate } from '../index';

describe('NotFoundTemplate', () => {
  it('to match snapshot', () => {
    const { container } = render(<NotFoundTemplate />);
    expect(container).toMatchSnapshot();
  });
});
