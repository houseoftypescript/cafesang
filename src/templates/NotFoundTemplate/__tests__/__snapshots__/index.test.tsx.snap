// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFoundTemplate to match snapshot 1`] = `
<div>
  <main
    class="h-screen"
  >
    <div
      class="MuiContainer-root MuiContainer-maxWidthLg h-full css-1oqqzyl-MuiContainer-root"
    >
      <div
        class="h-full flex flex-col items-center justify-center gap-4"
      >
        <h1
          class="text-9xl font-bold"
        >
          404
        </h1>
        <p
          class="uppercase text-3xl"
        >
          Page Not Found
        </p>
        <a
          href="/"
        >
          <button
            class="px-4 py-2 rounded bg-red-900 shadow-2xl text-white "
            type="button"
          >
            Go to Home Page
          </button>
        </a>
      </div>
    </div>
  </main>
</div>
`;
